### PR TITLE
Create a Paytm landing page clone

### PR DESCRIPTION
**Pull request by amanver45**
issue #840 
Closes: 

- **Title:** Create a Paytm landing page clone
- **Name:** Aman Verma
- **Idenitfy yourself:**  GSSOC, SSOC, JWOC, etc. Contributor

### Describe the add-ons or changes you've made 📃
Create a Paytm landing page clone

### video

https://github.com/user-attachments/assets/aebdd149-7aa3-4d36-9533-c4764753d39b

- [x] My code follows the code style of this project.
- [x] This change requires a documentation update

### Checklist: ☑️

- [x] My code follows the code style of this project.
- [x] My code follows the [Contributing Guidelines](https://github.com/Avdhesh-Varshney/WebMasterLog/blob/main/CONTRIBUTING.md) & [Code of Conduct](https://github.com/Avdhesh-Varshney/WebMasterLog/blob/main/CODE_OF_CONDUCT.md) of this project.
- [x] This PR does not contain plagiarized content.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly wherever it was hard to understand.
- [x] My changes generate no new warnings.

### CODE REVIEW

**Comment by amanver45** [comment]
@Avdhesh-Varshney I think this is level 2 issue  pls check the project  and reconsider it.

File: Vanilla-JS-Projects/Basic/Paytm Clone/readme.md
Comment (Avdhesh-Varshney) [inline Vanilla-JS-Projects/Basic/Paytm Clone/readme.md:1]: Update it carefully and rename ss file

**Review by Avdhesh-Varshney — CHANGES_REQUESTED** [review]
Review the guidelines properly

File: Vanilla-JS-Projects/Basic/Paytm Clone/readme.md
Comment (amanver45) [reply Vanilla-JS-Projects/Basic/Paytm Clone/readme.md:1]: Done

**Comment by amanver45** [comment]
Changes are done

**Review by Avdhesh-Varshney — CHANGES_REQUESTED** [review]
Folder name to be `Paytm-Clone`
Enlist your project in readme file.

**Comment by amanver45** [comment]
@Avdhesh-Varshney all changes are done check it

**Pull request closed without merging**